src/api.raml

```diff
@@ -1138,6 +1142,7 @@ traits:
                         Press package.
                     body:
                         application/vnd.elife.press-package+json;version=1:
+                            (deprecated):
```

Comment:
I _think_ this is valid RAML, but the JS library doesn't recognise it. (Making it a require string doesn't throw an error, but does if you move it directly inside `body`.)

Reply:
seems valid according to https://raml.org/developers/whats-new-raml-10 although they use `null` rather than `nil`

Reply:
They changed it to `nil` to avoid confusion with YAML's `null` type (https://github.com/raml-org/raml-spec/issues/514).